### ISSUE TITLE
Test plugin with old versions of Gatsby

### ISSUE THREAD

**Issue by ascorbic** [netlify/netlify-plugin-gatsby#373]
Currently the latest version of the plugin [requires Gatsby 4](https://github.com/netlify/plugins/blob/main/site/plugins.json#L515), but there's nothing specific that demands this. We should test this with Gatsby 3 and Gatsby 2 to ensure that a) it works with Gatsby Functions in Gatsby 3, and b) that in older versions it doesn't generate any functions. If this is confirmed, we should update plugins.json to reflect that

**Comment by nickytonline** [comment]
Is this something we still want to look at @ascorbic and if so, would it be reasonable to look at just v3, or should we still consider v2?

**Comment by MarcL** [comment]
@seldo - Do we have any data on which version of Gatsby people are using?

**Comment by seldo** [comment]
Sure do! https://app.mode.com/netlify/reports/00342b250700/runs/bfbe51689650

TLDR: in the last 30 days about 13k accounts built with Gatsby. Of those about half are on version 4; the next most popular version is 2, which about 25% of builds used. More details if you click through to Mode.

**Comment by orinokai** [comment]
Awesome, this is really useful. For reference, version 2 of Gatsby is no longer officially supported by Gatsby themselves https://www.gatsbyjs.com/docs/reference/release-notes/gatsby-version-support/

**Comment by seldo** [comment]
From this data supporting v3 is a no-brainer. Anybody still using v2 is probably not paying enough attention to care about whether we support them or not.

**Comment by taty2010** [comment]
Was able to get [this](https://github.com/taty2010/gatsby-v3-test) Gatsby v3 app to work locally using netlify dev, and functions are working as expected. @ascorbic if everything looks good on your end as well I can make the update for the plugins.json

**Comment by ascorbic** [comment]
Great!  Could you also try it with `netlify serve`?

**Comment by taty2010** [comment]
Everything is working when running `ntl serve`

**Comment by ascorbic** [comment]
Great! I'd say let's go ahead an update the package.json and [plugins.json](https://github.com/netlify/plugins/blob/main/site/plugins.json#L651)

**Comment by taty2010** [comment]
Updated plugin PR https://github.com/netlify/plugins/pull/1104
